Signal/src/view controllers/CallViewController.swift

```diff
@@ -472,7 +473,28 @@ class CallViewController: UIViewController, CallDelegate {
         case .answering:
             return NSLocalizedString("IN_CALL_SECURING", comment: "Call setup status label")
         case .connected:
-            return NSLocalizedString("IN_CALL_TALKING", comment: "Call setup status label")
+            if let call = self.call {
+                let callDuration = call.connectionDuration()
+                let callDurationDate = Date(timeIntervalSinceReferenceDate:callDuration)
+                let dateFormatter = DateFormatter()
```

Comment:
Forgive me if I'm going against norms established by this project but keeping formatting at a higher level might easy access to formatters for other models and VCs. Consider the example laid out [here](http://sandmoose.com/post/119962774817/caching-nsdateformatter) and a cautionary explanation on minimizing creation of formatters by bbum [here](http://stackoverflow.com/a/4442389). 

Creating a formatter cache would be out of scope for this PR so a separate PR may be advised.